modules/oe_bootstrap_theme_paragraphs/config/post_updates/00003/field.field.paragraph.oe_facts_figures.oe_bt_layout.yml

```diff
@@ -0,0 +1,24 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.oe_bt_layout
+    - paragraphs.paragraphs_type.oe_facts_figures
+id: paragraph.oe_facts_figures.oe_bt_layout
+field_name: oe_bt_layout
+entity_type: paragraph
+bundle: oe_facts_figures
+label: 'Layout'
```

Comment:
In the edit form I would expect a label like "Number of columns", not "Layout".
But if this was already decided by others, then what can I say..

Reply:
In fact this field was formerly the 'variant' field. https://github.com/openeuropa/oe_bootstrap_theme/pull/123/commits/41a02e6d538438fdd749bff4d8a746ac963b461e

To check if is worth the time of changing the name again.